### PR TITLE
Feature/add to outfit (review 2nd)

### PR DESCRIPTION
**Pull request by acdavitt**
@julia-thea @ChrisRPeterson @dylanreid7 

*** NOTE: this branch is branched off of feature/outfit, so please review feature/outfit first and merge before merging this branch ***
--When the Add to Outfit Card is clicked, the current product is added to the outfit list.  
--If the current style has already been added to the outfit list, an alert will pop-up preventing it from being added twice

### CODE REVIEW

File: client/components/RelatedProducts/RelatedProducts.jsx [hunk 1]
```diff
@@ -47,13 +55,37 @@ const RelatedProducts = ({productId}) => {
...
+        sale_price: currentStyle.sale_price,
+        photo: currentStyle.photos[0].url,
+        style_id: currentStyle.style_id,
+        category: product.category,
+        rating: averageRating
+      }
+      outfitCardsDataCopy.push(outfitCardData)
+      setOutfitCardsData(outfitCardsDataCopy)
+    } else {
+      alert('This style has already been added to your outfit list!')
```
Comment (ChrisRPeterson) [inline client/components/RelatedProducts/RelatedProducts.jsx:77]: I think the alert is fine for now, but in the future, I think you might be able to use some material-ui component to do something similar.

File: client/components/RelatedProducts/RelatedProducts.jsx [hunk 2]
```diff
@@ -47,13 +55,37 @@ const RelatedProducts = ({productId}) => {
...
+  }
+
   return (
     <div id='related'>
       <Grid container spacing={2}>
         <RelatedProductsList relatedProductsData={relatedProductsData} />
       </Grid>
       <Grid>
-        <CustomOutfitList />
+      {console.log({outfitCardsData})}
```
Comment (ChrisRPeterson) [inline client/components/RelatedProducts/RelatedProducts.jsx:87]: Can you remove the console log before we merge?

**Review by ChrisRPeterson — CHANGES_REQUESTED** [review]
Looks amazing! I messed around with it for a bit changing styles and stuff, the custom styles are working really well.

File: client/components/RelatedProducts/RelatedProducts.jsx (hunk 1 above)
Comment (acdavitt) [reply client/components/RelatedProducts/RelatedProducts.jsx:77]: ooo good idea.  Didn't even think about that.  I'll investigate and create a separate ticket for it

File: client/components/RelatedProducts/RelatedProducts.jsx (hunk 2 above)
Comment (acdavitt) [reply client/components/RelatedProducts/RelatedProducts.jsx]: @ChrisRPeterson Done!

**Review by ChrisRPeterson — APPROVED** [review]

**Pull request merged**